package.json

```diff
@@ -28,17 +28,21 @@
   "homepage": "https://github.com/elasticio/sugarcrm-component#readme",
   "dependencies": {
     "crc": "3.2.1",
-    "elasticio-node": "0.0.8",
-    "elasticio-sailor-nodejs": "^2.2.0",
-    "request": "^2.81.0",
-    "underscore": "^1.8.3"
+    "elasticio-node": "0.0.9",
+    "elasticio-rest-node": "1.2.3",
+    "elasticio-sailor-nodejs": "2.5.1",
```

Comment:
In order to publish the component with the latest version of the sailor SDK, I'd recommend to upgrade the `elasticio-sailor-nodejs` dependency to `2.5.4`

Reply:
I checked npm and can see you have even fresher version, so updated to 2.6.0

Reply:
I checked npm and can see you have even fresher version, so updated to 2.6.0